### ISSUE TITLE
Cloudflare Settings to prevent a little bit from DDOS

### ISSUE THREAD

**Issue by Marow** [sammy007/open-ethereum-pool#377]
I tried to Prevent a little bit from DDOS with Cloudflare and hide my IP Address, cause we get the last Days DDOS directly to our IP.

We can get a new IP Address if we want, but first i want to setup the Cloudflare Settings.

I setup that, but its not working good. Only the small miners get access after a time. Big miners get kicked out. 

Someone has Cloudflare Firewall Settings or something for Mining Pool?

**Comment by mikeyb** [comment]
[READ](https://github.com/sammy007/open-ethereum-pool#configuration)

**Comment by Marow** [comment]
@mikeyb 

I mean Settings IN CloudFlare if someone has working with it.
I know that Setting in Open-Ethereum-Pool but i dont know, how that can be work (with Cloudflare) cause Cloudflare kicked out Miners on my Pool.

So i ask, if someone run with Cloudflare, HOW and What Settings on Cloudflare they use

**Comment by mikeyb** [comment]
You most likely need enterprise account with CF, otherwise nothing in the headers gets forwarded correctly

**Comment by sammy007** [comment]
Yeah CF is a BS without enterprise plan, you don't have to use their HTTP filters, it's designed for http web servers. Grab a server on online.net if you need european DC.
